src/aurora/erfwriter.cpp

```diff
@@ -0,0 +1,113 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  Writing BioWare's ERFs (encapsulated resource file).
+ */
+
+#include <ctime>
+
+#include "src/aurora/erfwriter.h"
+
+namespace Aurora {
+
+static const uint32 kLocStringTableOffset = 160; // 160 is the size of the erf header.
```

Comment:
You didn't need to make it a global. It should still be in the scope of the constructor, defined right before it's used.